assets/dev/scss/frontend/_container.scss

```diff
@@ -53,6 +53,7 @@
 	z-index: var( --z-index );
 	overflow: var( --overflow );
 	transition: var( --transition );
+	isolation: isolate;
```

Comment:
Possibly you can add a comment to explain what this does and a reference to the Jira ticket (ED-7944).

Reply:
I'm seeing a lot of troubles with `isolation: isolate;` . It creates a new stack and completely isolates the element. Can possibly lead to issues with z-indexes and transforms.